entrypoint.sh

```diff
@@ -57,11 +58,16 @@ fi
 # Add changes and push commit
 git add .
 
+commit_signoff=""
+if [ "${GIT_PUSH_SIGN_OFF}" = "true" ]; then
```

Comment:
I think this should be `GIT_COMMIT_SIGN_OFF` instead

Reply:
You are correct, copy/paste issue!